packages/plugins/plugin-deck/src/components/DeckLayout/ComplementarySidebar.tsx

```diff
@@ -23,15 +30,57 @@ export type ComplementarySidebarProps = {
   flatDeck?: boolean;
 };
 
-export const ComplementarySidebar = ({ context, layoutParts, flatDeck }: ComplementarySidebarProps) => {
+const panels = ['comments', 'settings'] as const;
+type Panel = (typeof panels)[number];
+const getPanel = (part?: string): Panel => {
+  if (part && panels.findIndex((panel) => panel === part) !== -1) {
+    return part as Panel;
+  } else {
+    return 'settings';
```

Comment:
Consider `log/warning` for unknown value?

Reply:
I'm specifically expecting this happen as an initial state in some cases because of cruft in the cached deck layout data so I think it's better for it to just fall through. If we find it harmful I wouldn't mind having a log there eventually, but would prefer to be conservative about what we're firing into Sentry for now.